src/api.py

```diff
@@ -0,0 +1,18 @@
+import requests 
+import hashlib
+def fetch(url):
+    con = requests.head(url)
+    r = requests.get(url)
+    name = url.split('/')[-1]
```

Comment:
Are you sure the last segment is the name? and always? what if I have a URL that ends with '/' ? the name will be empty.

Reply:
Got this one